Tasks/VsTestPlatformToolInstallerV1/vstestplatformtoolinstaller.ts

```diff
@@ -47,6 +52,15 @@ async function startInstaller() {
                 await new NugetFeedInstaller(consolidatedCiData)
                     .getVsTestPlatformToolFromSpecifiedFeed(packageSource, testPlatformVersion, versionSelectorInput, null);
                 break;
+
+            case 'customfeed':
+                await new NugetFeedInstaller(consolidatedCiData)
```

Comment:
why are we passing around consolidatedCiData object?
From Object principles why should nugetfeedInstaller require "CI" object?

Reply:
how do i replicate a static singleton class in java script?